src/components/molecules/Footer/Footer.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+import { HOMEPAGE_URL, TERMS_AND_CONDITIONS_URL } from "settings";
+
+export const Footer = () => (
+  <div className="footer-wrapper">
+    <a className="terms-and-conditions" href={TERMS_AND_CONDITIONS_URL}>
+      Terms & Conditions
+    </a>
+    <div className="footer-separator">|</div>
+    <div>
+      Made with{" "}
+      <span role="img" aria-label="heart">
+        ❤️
+      </span>
+      <a className="powered-by" href={HOMEPAGE_URL}>
+        powered by Sparkle
+      </a>
+      <span role="img" aria-label="sparkle">
```

Comment:
Great to see accessibility being considered here

Reply:
🎉 

This is something I would like to add to our lint/test steps in future, there are a few good tools we can use, but i'm scared about what they will tell us we need to fix for now.. :p